### PR TITLE
update ocp to 4.12 [odf-console CI]

### PR DESCRIPTION
**Pull request by SanjalKatiyar**

### CODE REVIEW

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 1]
```diff
@@ -50,23 +50,28 @@ tests:
...
       dependencies:
       - env: ODF_CONSOLE_FULL_IMG_NAME
         name: odf-console-img
       from: src
       resources:
         requests:
-          cpu: 100m
-          memory: 200Mi
+          cpu: 1000m
+          memory: 1Gi
```
Comment (bipuladh) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:74]: This is only increasing the memory and CPU of the pod used to run cypress on. This is not required.

Comment (rexagod) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:74]: ACK.

Comment (SanjalKatiyar) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:74]: yeah, I was trying out few things earlier, missed removing it later...

**Pull request merged**